### PR TITLE
[evaluation] improvement on evaluation

### PR DESCRIPTION
**Pull request by chengeharrison**
## 📌 Checklist before creating the PR

- [ ] I have created an issue for this PR for traceability
- [x] The title follows the standard format: `[doc/gemini/tensor/...]: A concise description`
- [x] I have added relevant tags if possible for us to better distinguish different PRs

## 🚨 Issue number

> Link this PR to your issue with words like fixed to automatically close the linked issue upon merge
>
> e.g. `fixed #1234`, `closed #1234`, `resolved #1234`

## 📝 What does this PR do?

> Summarize your work here.
> if you have any plots/diagrams/screenshots/tables, please attach them here.

1. fix a bug when the config file contains one category but the answer file doesn't contains that category
2. fix Chinese prompt file
3. support gpt-3.5-turbo and gpt-4 evaluation
4. polish and update README

## 💥 Checklist before requesting a review

- [ ] I have linked my PR to an issue ([instruction](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue))
- [ ] My issue clearly describes the problem/feature/proposal, with diagrams/charts/table/code if possible
- [x] I have performed a self-review of my code
- [x] I have added thorough tests.
- [x] I have added docstrings for all the functions/methods I implemented

## ⭐️ Do you enjoy contributing to Colossal-AI?

- [x] 🌝 Yes, I do.
- [ ] 🌚 No, I don't.

Tell us more if you don't enjoy contributing to Colossal-AI.

### CODE REVIEW

File: applications/Chat/evaluate/prompt/evaluation_prompt/evaluation_prompt_cn.json
Comment (Camille7777) [inline applications/Chat/evaluate/prompt/evaluation_prompt/evaluation_prompt_cn.json:1]: Hi, yuanchen, why there are some newlines in this file?

File: applications/Chat/evaluate/evaluator.py [hunk 1]
```diff
@@ -74,17 +80,22 @@ def switch(metric):
...
+            category_metrics = self.params[category]["GPT"]
 
             prompt = self.gpt_evaluation_prompt.get(category, None)
             if prompt is None:
                 print(f"No prompt for category {category}! Use prompt for category general now.")
                 prompt = self.gpt_evaluation_prompt["general"]
 
-            self.gpt35_evaluation_results[category] = gpt_evaluate.gpt35_evaluate(answers_per_category[category],
-                                                                                  prompt, category_metrics, category)
+            self.gpt_evaluation_results[category] = gpt_evaluate.gpt_evaluate(answers_per_category[category], prompt,
```
Comment (TongLi3701) [inline applications/Chat/evaluate/evaluator.py:96]: `gpt_evaluate.gpt_evaluate` is a bit confusing. Please rename it to `gpt_evaluate.evaluate` or other function names that shows the differences.

File: applications/Chat/evaluate/README.md [hunk 2]
```diff
@@ -4,92 +4,102 @@ In this directory, we introduce how you can evaluate your model with our pipelin
...
+|       Diversity       | 多样性(1-5)：答案使用语言是否优美，具有有一定的创造性和想象力。然而，回答也应该保持合理和适度，不要过于夸张或离题。 | 1. 仔细阅读整个回答，确保完全理解回答所表达的内容和主题。<br/> 2. 在阅读回答的同时，注意语言的质量，例如措辞是否正确，语言是否生动等。<br/> 3. 检查回答的创造性和想象力，看看回答是否能够吸引人阅读下去。<br/> 4. 检查回答的合理性和适度，看看回答是否夸张或离题。5. 将多样性的评分打分在1到5之间，5分表示回答的质量很好，能够吸引人阅读，1分表示回答的内容生硬或者有离题的问题。 |
+|       Fidelity        | 保真度(1-5)：答案是否能够严格遵守角色的设定回答给定的请求。  | 1. 仔细阅读问题，了解角色在问题中的设定和表现，包括职业、背景、观点、性格等方面。<br/> 阅读题目的请求，确认回答请求时需要注意的细节。<br/> 3. 对比提供的回答与该角色的设定，评估回答是否能够严格遵守角色的设定。<br/> 4. 结合以上评估结果给出保真度的评分，范围从1到5分，其中1分表示回答与角色设定完全不符，5分表示回答完全符合角色设定且满足给定请求。 |
+|      Conciseness      | 简明扼要(1-5)：答案是否简明扼要，没有冗余内容。              | 1. 阅读题目，提取出材料的重点。<br/> 2. 阅读该总结，并注意其中的主要观点和信息。<br/> 3. 评估总结的长度。一个简明扼要的总结通常应该在几句话或几段文字内传达关键信息，而不是冗长的段落或文章。<br/> 4. 检查总结是否包含与主要观点无关的信息或冗余信息。<br/> 5. 确定总结涵盖了材料中的关键信息，并且没有忽略任何重要细节。<br/> 6. 给总结打出1-5的分数，其中5表示总结简明扼要，没有冗余内容，而1表示总结冗长或包含不必要的信息，难以理解或记忆。根据您的判断，打出适当的得分。 |
 
 GPT-3.5 evaluates the quality of model predictions based on the given prompt words and gives a score between 1-5.
```
Comment (TongLi3701) [inline applications/Chat/evaluate/README.md:77]: Here should be GPT models.

**Review by TongLi3701 — APPROVED** [review]
Thanks Yuanchen.

Overall looks good. Some minor comments. 
Please also test locally make sure everything is runnable including GPT eval and Automatic eval.

Please make all table headers centre such as `Description`, `Chinese Example` and so on.

**Review by Camille7777 — APPROVED** [review]

**Review by TongLi3701 — APPROVED** [review]
Thanks Yuanchen.

**Pull request merged**